GPS.Data/Validate.cs

```diff
@@ -0,0 +1,176 @@
+﻿using GPS.Data.ParserObjects;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace GPS.Data
+{
+	public class Validate
```

Comment:
This would be a good candidate to make extension methods since there is no instance members here.  This way you can just call .Validate off the existing object